service/src/main/java/uk/gov/hmcts/reform/fpl/service/OrderValidationService.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+
+import java.util.List;
+
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static org.springframework.util.CollectionUtils.isEmpty;
+import static uk.gov.hmcts.reform.fpl.enums.OrderStatus.SEALED;
+
+@Service
+public class OrderValidationService {
+    public List<String> validate(final CaseData caseData) {
+        if (SEALED == caseData.getStandardDirectionOrder().getOrderStatus()
+            && isEmpty(caseData.getHearingDetails())) {
+            return singletonList(
```

Comment:
I think it is more of a convention to use `List.of` instead of `Collections.singletonList`

Reply:
went with `Collections.singletonList` as we are just returning one string and `List.of` will be preferred if we have more than one string/object.